include/sympack/symPACKMatrix2D.hpp

```diff
@@ -5,8 +5,18 @@
 #include "sympack/symPACKMatrixBase.hpp"
 #include "sympack/symPACKMatrix.hpp"
 
+
 #include "sympack/mpi_interf.hpp"
 
+#ifdef CUDA_MODE
+#include "cuda_runtime.h"
+#include "device_launch_parameters.h"
+#include "cublas_v2.h"
+#include "cusolverDn.h"
+#endif
+
+#include <omp.h>
```

Comment:
This include appears to be have added for no reason (and unconditionally), and runs at least a small risk of introducing a conflict with some OpenMP implementations.

```suggestion
```